table/table_test.cc

```diff
@@ -2604,16 +2604,44 @@ TEST_F(GeneralTableTest, ApproximateOffsetOfCompressed) {
 }
 
 TEST_F(HarnessTest, Randomized0) {
-  // part 1 out of 2
+  // part 1 out of 4
   const size_t part = 1;
-  const size_t total = 2;
+  const size_t total = 4;
   RandomizedHarnessTest(part, total);
 }
 
 TEST_F(HarnessTest, Randomized1) {
-  // part 2 out of 2
+  // part 2 out of 4
   const size_t part = 2;
-  const size_t total = 2;
+  const size_t total = 4;
+  RandomizedHarnessTest(part, total);
+}
+
+TEST_F(HarnessTest, Randomized5) {
```

Comment:
Where is Randomized 2 3 and 4?

Reply:
naming is a bit off, maybe rename randomized0 to randomized1n2, randomized1 to randomized3n4?